src/view.tree.js

```diff
@@ -30,31 +30,33 @@ class TreeView {
     const $jstree = this.$jstree;
 
     $jstree.settings.core.data = (node, cb) => {
```

Comment:
?

```suggestion
    $jstree.settings.core.data = async (node, cb) => {
```

Reply:
same as the callback inside `done' of ajax, this function can't be an async one

Reply:
Please help put comment for reason why to use anonymous execution